charts/timed/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: timed
 description: Chart for Timed application
 type: application
-version: 0.4.10
-appVersion: v1.1.2
+version: 0.4.11
+appVersion: v1.1.3
```

Comment:
Why did you bump the appVersion?

Reply:
because all app needed a version bump after changing the dependency of helm-docs (generating new README.md)

Reply:
oh my bad. AppVersion does not need to be bumped... let me fix that

Reply:
6f63f9d fixed this

Reply:
Chart version yes...